shuup/core/models/_orders.py

```diff
@@ -558,17 +567,31 @@ def create_refund(self, refund_data, created_by=None):
             if amount > self.taxful_total_price.amount:
                 raise RefundExceedsAmountException
 
+            # If restocking products, calculate quantity of products to restock
+            product = parent_line.product
+            if (restock_products and quantity and product and (product.stock_behavior == StockBehavior.STOCKED)):
+                from shuup.core.suppliers.enums import StockAdjustmentType
```

Comment:
Note: I was still getting a circular import error here, so this seems like it might be necessary